src/app/src/main/kotlin/org/imperial/mrc/hint/models/AdrResource.kt

```diff
@@ -0,0 +1,3 @@
+package org.imperial.mrc.hint.models
+
+data class AdrResource(val url: String, val id: String? = "", val resourceId: String? = "")
```

Comment:
Do we need `url`? I don't think it actually get used, does it? Because you're constructing the resource url from the other bits, rather than using the more generic url that we have in the front end.

Reply:
It's used for accessing ADR endpoint, needed.

Reply:
Ah yes, it's used to get the original filename.